docs/en/ingest-management/getting-started-traces.asciidoc

```diff
@@ -0,0 +1,232 @@
+**Create user and enable central management**.
++
+[role="screenshot"]
+image::images/kibana-fleet-enable.png[{fleet} showing prompt to enable central management]
+
+[discrete]
+[[add-agent-to-fleet-traces]]
+== Step 2: Add an {agent} to {fleet}
+
+[TIP]
+====
+If you plan on enabling Real User Monitoring (RUM), you must run {agent} centrally.
+If RUM is disabled, {agent} should be run on edge machines.
+====
+
+include::elastic-agent/elastic-agent.asciidoc[tag=agent-install-intro]
+
+To send APM data to the {stack}:
+
+. On the **Agents** tab in {fleet}, click **Add agent**, and look at the
+deployment instructions under **Enroll in {fleet}**.
+
+. As instructed, download and extract the {agent} to your host. To do this
+quickly from the command line, run:
++
+--
+include::elastic-agent/install-elastic-agent.asciidoc[tag=install-elastic-agent]
+
+See the https://www.elastic.co/downloads/elastic-agent[download page] for other
+installation options.
+--
+
+. Back in {fleet}, under **Choose an agent policy**, notice that the default
+policy is selected. The default policy includes a system integration for
+collecting logs and metrics from the host system. Use the default policy to get
+started quickly.
+
+. Under **Enroll and start the {agent}**, copy the install command if it's
+available for your platform, or make a note of the {kib} URL and enrollment
+token that was generated by {fleet}.
++
+[role="screenshot"]
+image::images/kibana-fleet-enroll.png[{fleet} showing agent enrollment page]
+
+. From the agent directory, run the appropriate command to install, enroll, and
+start an {agent}. Note that this command installs {agent} files in the locations
+described in <<installation-layout>>.
++
+--
+include::{tab-widgets}/install-widget.asciidoc[]
+--
++
+Because {agent} is installed as an auto-starting service, it will restart
+automatically if the system is rebooted.
+
+. In {fleet}, click **Continue** to go to the **Agents** tab, where you should
+see the newly enrolled agent.
++
+[role="screenshot"]
+image::images/kibana-fleet-agents.png[{fleet} showing enrolled agents]
+
+TIP: If the status hangs at Enrolling, make sure the `elastic-agent` process
+is running.
+
+If you run into problems, see <<fleet-troubleshooting>>.
+
+[discrete]
+[[add-apm-integration]]
+== Step 3: Add the APM integration
+
+Next, you'll browse a catalog of integrations, then add an APM integration to
+the default policy used by your agent. You use policies to manage settings
+across a group of agents. An agent policy may contain any number of integrations
+for collecting observability data from the various services running on your
+host.
+
+. In {kib}, go back to **Management > {fleet}**, and click the **Integrations**
+tab. Use the search bar to find the APM integration.
++
+[role="screenshot"]
+image::images/kibana-fleet-integrations-apm.png[{fleet} showing APM integration]
+. Click the APM integration to see more details about it, then click
+**Add APM**.
```

Comment:
```suggestion
**Add Elastic APM**.
```

Reply:
Does the button have Elastic in it? I'll look again, but I'm pretty sure it only says "Add APM"

Reply:
Aha, I'm wrong!